nustyle.css

```diff
@@ -376,3 +384,204 @@ ul.icons2 {
   align-self: center;
   margin-top: 15px;
 }
+
+#form-button:hover {
+  background-color: #f8b791;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+#footer-hr {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  #nav-mobile {
+    display: none;
+  }
+
+  #nav-desktop {
+    display: inline;
+    align-self: center;
+  }
+
+  #nav-desktop a {
+    padding-right: 32px;
+    margin: 0;
+    font-family: Inter;
+    font-size: 15px;
+    font-weight: 600;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  header img {
+    height: 18px;
+    padding-bottom: 0;
+    padding-right: 119px;
+  }
+
+  .header-logo {
+    padding-left: 154px;
+    font-size: 20px;
+    padding-bottom: 26px;
+  }
+
+  .container {
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+    background-image: url(./assets/Illustration-header.png);
+    background-position: right;
+  }
+
+  .icons {
+    flex-direction: column;
+    padding: 279px 0 21px 36px;
+  }
+
+  .intro {
+    padding-left: 71px;
+  }
+
+  .intro p {
+    font-family: 'Poppins';
+    font-size: 16px;
+    font-weight: 400;
+    padding-right: 731px;
+    margin-bottom: 251px;
+  }
+
+  .intro h1 {
+    padding-top: 192px;
+  }
+
+  .icons li {
+    padding-bottom: 21px;
+  }
+
+  .works {
+    margin: 135px 153.5px 0 153.5px;
+  }
+
+  .main-project {
+    padding: 0;
+  }
+
+  .hr {
+    display: inline-block;
+    content: "";
+    background-color: #212121;
+    height: 1px;
+    width: 61%;
+    padding-right: 30px;
+    margin-left: 20px;
+  }
+
+  .banner {
+    border: none;
+    display: flex;
+    align-items: center;
+  }
+
+  #project {
+    display: grid;
+    grid-template-columns: 600px 400px;
+    gap: 24px;
+  }
+
+  #project-story {
+    width: min-content;
+  }
+
+  #project img {
+    min-height: 400px;
+    min-width: 600px;
+    display: flex;
+    flex-shrink: inherit;
+  }
+
+  #project h2 {
+    margin-top: 0;
+  }
+
+  #projects-container {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    gap: 29px;
+    margin-top: 24px;
+    margin-bottom: 130px;
+  }
```

Comment:
- [ ] Dear @Nuga0909, You did great aligning projects on the screen, But there are some details missing.
1. `See Project` Button should only be visible when a user hovers over a project.
2. The description and dark opacity should disappear when the mouse hovers over a project.

## Your design
![your-card](https://user-images.githubusercontent.com/69077061/199803294-48faaeeb-625d-4889-a01c-6037bdb7cb54.PNG)

## Figma design
![figma-cards](https://user-images.githubusercontent.com/69077061/199803344-ff63fd9d-69ea-47b2-bee3-9b1191f70036.PNG)